constants.js

```diff
@@ -53,6 +56,7 @@ const respondToReviewCheck = 'respond-to-review-check';
 const oldBuildLabelCheck = 'old-build-label-check';
 
 const checksWhitelist = {
+  // eslint-disable-next-line quote-props
```

Comment:
Why this needs to be added?

Reply:
Reason mentioned here: https://github.com/oppia/oppiabot/pull/233#discussion_r602789862